server/schemas/thread.graphql

```diff
@@ -0,0 +1,35 @@
+type Thread {
+  id: ID!
+  comments: [Comment]
+}
+
+type Comment {
+  id: ID!
+  authorId: ID!
+  content: String!
```

Comment:
```suggestion
  content: String!
  createdAt: DateTime!
```

createdAt can use a timestamp of a commnet ID, so there is no need to add a `createdAt` field to the domain model.